src/components/Global/Footer/index.tsx

```
@@ -1,17 +1,15 @@
 'use client'
+
 import Link from 'next/link'
-import { useColorMode } from '@chakra-ui/color-mode'
 
 import * as _consts from './consts'
 
 type FooterProps = {}
 
 const Footer = ({}: FooterProps) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)